azure/.changeset/config.json

```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+	"changelog": [
+		"changesets-format-with-issue-links",
+		{
+			"repoBaseUrl": "https://github.com/microsoft/FluidFramework",
```

Comment:
Sanity check: what is this used for? Should this point to the azure dir?

Reply:
It's used to generate links to issues and PRs in the changelog, so this is correct.